### DISCUSSION TITLE
installation fails in macOS 10.15.7

### DISCUSSION THREAD

**Discussion by l4l0** [openvinotoolkit/openvino_notebooks#38] [Q&A] [answered]
Hi all,

I am trying to install openvino notebooks in my iMac running macOS 10.15.7, but installation step 4 fails, giving me the following error:

Building wheels for collected packages: editdistance
  Building wheel for editdistance (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/edrom/Documents/openvino_notebooks/openvino_env/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/db/nzljqgts42n3z2l9zzrngx500000gn/T/pip-install-axu5bm_g/editdistance/setup.py'"'"'; __file__='"'"'/private/var/folders/db/nzljqgts42n3z2l9zzrngx500000gn/T/pip-install-axu5bm_g/editdistance/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/db/nzljqgts42n3z2l9zzrngx500000gn/T/pip-wheel-ejemxe5p
       cwd: /private/var/folders/db/nzljqgts42n3z2l9zzrngx500000gn/T/pip-install-axu5bm_g/editdistance/

There is a lot more lines of error following the above one but dont know if they are relevant or not. I am attaching a log file with the output I got. Hope someone can help me to find out whats going wrong.

Regards,

Eduardo.
[install_log.txt](https://github.com/openvinotoolkit/openvino_notebooks/files/6360347/install_log.txt)

**Comment by helena-intel** [comment, marked as answer]
Yikes! Thanks for reporting the issue and attaching the log!  In the logs I see "#error architecture not supported" . That is similar to this issue reported on StackOverflow: https://stackoverflow.com/questions/64364310/how-do-i-resolve-error-architecture-not-supported-during-pip-install-psutil The answer there suggests that prepending the pip install command with `ARCHFLAGS="-arch x86_64"` should work. Could you try that? Instead of `pip install openvino-dev` you would then do `ARCHFLAGS="-arch x86_64" pip install openvino-dev`

**Reply by l4l0** [reply]
Hi Helena,
Thanks for your swift response!! I will try your suggestion and report back any outcome.
Regards,
Eduardo.

**Reply by l4l0** [reply]
Hi Helena,
It worked like a charm!!
Regards,
Eduardo.

**Reply by helena-intel** [reply]
Thanks for reporting back, I am glad it worked!

**Comment by raymondlo84** [comment]
Wow upvoting this!!! (@ryanloney - documentation can be updated to say it works on 10.15.7 with that new pip install command)
